<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-KB6UDKRL" failures="5" tests="10" name="TestRunner-chrome" time="104.892" errors="0" timestamp="2021-10-18T10:21:59 IST">
  <testcase classname="scriptsContacts.TC_12_createcontact_assignedTo_Teamselling_Group" name="createcontact_assignedTo_Teamselling_Group" time="5.238"/>
  <testcase classname="scriptsContacts.TC_12_createcontact_assignedTo_Teamselling_Group" name="createcontact_assignedTo_Teamselling_Group" time="4.804"/>
  <testcase classname="scriptsContacts.TC_12_createcontact_assignedTo_Teamselling_Group" name="createcontact_assignedTo_Teamselling_Group" time="4.469"/>
  <testcase classname="scriptsContacts.TC_12_createcontact_assignedTo_Teamselling_Group" name="createcontact_assignedTo_Teamselling_Group" time="4.291"/>
  <testcase classname="scriptsContacts.TC_12_createcontact_assignedTo_Teamselling_Group" name="createcontact_assignedTo_Teamselling_Group" time="4.443"/>
  <testcase classname="scriptsContacts.TC_11_createcontact_assignedTo_Support_Group" name="createcontact_assignedTo_Support_Group" time="4.856">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at scriptsContacts.TC_11_createcontact_assignedTo_Support_Group.createcontact_assignedTo_Support_Group(TC_11_createcontact_assignedTo_Support_Group.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- createcontact_assignedTo_Support_Group -->
  <testcase classname="scriptsContacts.TC_11_createcontact_assignedTo_Support_Group" name="createcontact_assignedTo_Support_Group" time="5.301">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at scriptsContacts.TC_11_createcontact_assignedTo_Support_Group.createcontact_assignedTo_Support_Group(TC_11_createcontact_assignedTo_Support_Group.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- createcontact_assignedTo_Support_Group -->
  <testcase classname="scriptsContacts.TC_11_createcontact_assignedTo_Support_Group" name="createcontact_assignedTo_Support_Group" time="5.211">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at scriptsContacts.TC_11_createcontact_assignedTo_Support_Group.createcontact_assignedTo_Support_Group(TC_11_createcontact_assignedTo_Support_Group.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- createcontact_assignedTo_Support_Group -->
  <testcase classname="scriptsContacts.TC_11_createcontact_assignedTo_Support_Group" name="createcontact_assignedTo_Support_Group" time="5.224">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at scriptsContacts.TC_11_createcontact_assignedTo_Support_Group.createcontact_assignedTo_Support_Group(TC_11_createcontact_assignedTo_Support_Group.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- createcontact_assignedTo_Support_Group -->
  <testcase classname="scriptsContacts.TC_11_createcontact_assignedTo_Support_Group" name="createcontact_assignedTo_Support_Group" time="5.289">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at scriptsContacts.TC_11_createcontact_assignedTo_Support_Group.createcontact_assignedTo_Support_Group(TC_11_createcontact_assignedTo_Support_Group.java:73)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- createcontact_assignedTo_Support_Group -->
</testsuite> <!-- TestRunner-chrome -->
